### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ddodoo**
Added Scale Capacity and Deimos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,6 +79,10 @@ This repository house companies who are known to hire directly from Africa and s
 
 39. <h4><a href="https://energi.world/join-our-team/">Energi</a></h4><p>Energi (NRG) is a next-generation Proof-of-Stake (PoS) cryptocurrency that combines smart contracts, decentralized governance, and a self-funding treasury. </p><p><strong>Location- </strong>British Virgin Island | <small>Remote</small> </p>
 
+40.  <h4><a href="https://deimos.io/">Deimos</a></h4><p> Cloud Native Developer and Security Operations Company. </p><p><strong>Location- </strong>South Africa-Cape Town | <small>Remote(South Africa, Kenya, Nigeria, Ghana)</small> </p>
```
Comment (saviour123) [inline README.md:82]: We need the link to its carreer page. @ddodoo

File: README.md [hunk 2]
```diff
@@ -79,6 +79,10 @@ This repository house companies who are known to hire directly from Africa and s
...
+
+41.  <h4><a href="https://scalecapacity.com/">Scale Capacity</a></h4><p> Amazon Web Services, Cloud, Information Technology, Public Cloud, Cloud Migration, Cloud Optimization, Consulting, AWS, IaaS, IT Consulting, Cloud Consulting, Staffing, and Digital Transformation. </p><p><strong>Location- </strong> USA-Los Angeles, CA | <small>Global Remote</small> </p>
```
Comment (saviour123) [inline README.md:84]: The link to its carreer page or hiring manager email. Not the website

**Review by saviour123 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ddodoo) [reply README.md]: @saviour123  I have updated the URL link.

Comment (ddodoo) [reply README.md]: Recent commit has the changes

**Review by saviour123 — APPROVED** [review]

**Pull request merged**